packages/lib/package.json

```diff
@@ -39,7 +39,7 @@
         "typescript": "^4.4.2"
     },
     "scripts": {
-        "build:schema": "yarn ts-json-schema-generator -f tsconfig.json --path src/lib/Messaging.ts --type Message -o src/schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Messages.ts --type Acknoledgment -o src/delivery/schema/Acknoledgment.schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/messaging/Messaging.ts --type EncryptionEnvelop -o src/messaging/schema/EncryptionEnvelop.schema.json --no-type-check  && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Delivery.ts --type DeliveryServiceProperties -o src/delivery/schema/DeliveryServiceProperties.schema.json --no-type-check",
+        "build:schema": "yarn ts-json-schema-generator -f tsconfig.json --path src/lib/Messaging.ts --type Message -o src/schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Messages.ts --type Acknoledgment -o src/delivery/schema/Acknoledgment.schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/messaging/Messaging.ts --type EncryptionEnvelop -o src/messaging/schema/EncryptionEnvelop.schema.json --no-type-check  && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Delivery.ts --type DeliveryServiceProperties -o src/delivery/schema/DeliveryServiceProperties.schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Session.ts --type Session -o src/delivery/schema/Session.schema.json --no-type-check",
```

Comment:
do we already have an issue for making this shorter?

Reply:
I can create one